test/tools/docker-tests/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Example usage
+#
+# $ cd test/tools/docker-tests
+# $ docker build --tag test-node-mongodb-native .
+# $ docker run -it --net=host -v /tmp:/tmp test-node-mongodb-native 3.5 argon
+
+FROM centos:7 AS base
+RUN yum update -y && yum install -y git make gcc gcc-c++
+
+FROM base
+
+WORKDIR /root
+COPY entrypoint.sh entrypoint.sh
+
+ENV MONGODB_UNIFIED_TOPOLOGY 1
+ENV MONGODB_HOST "host.docker.internal"
```

Comment:
proposed environment variables:
- `MONGODB_URI` - the connection string (matches our testing environment)
- `MONGODB_DRIVER_VERSION` - version of the driver to install
- `NODE_VERSION` - the version of node to install